astropy/cosmology/tests/test_flrw.py

```diff
@@ -1080,12 +1128,21 @@ def test_clone_change_param(self, cosmo):
             else:
                 assert u.allclose(v, getattr(cosmo, n), atol=1e-4 * getattr(v, "unit", 1))
 
+    # @pytest.mark.parametrize("z", valid_zs)
```

Comment:
Why these commented out lines?

Reply:
I plan to return to these. The original test is comparing against precomputed cases. Here I'm concentrating on transferring tests. Once the old test suite is migrated I want to switch to a uniform set of `z` and recompute the tests.